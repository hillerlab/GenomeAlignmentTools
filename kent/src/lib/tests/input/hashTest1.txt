apple apple1
cat cat1
dog dog1
elephant elephant1
frog frog1
frog frog2
frog frog3
frog frog4
girl girl1
house house1
infix infix1
jail jail1
kite kite1
kite kite2
kite kite3
lamb lamb1
mother mother1
nana nana1
nana nana2
ostrich ostrich1
pool pool1
yak yak1
yak yak2
zebra zebra1
