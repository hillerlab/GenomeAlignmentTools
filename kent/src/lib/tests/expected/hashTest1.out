apple=apple1
cat=cat1
dog=dog1
elephant=elephant1
frog=frog1
frog=frog2
frog=frog3
frog=frog4
girl=girl1
house=house1
infix=infix1
jail=jail1
kite=kite1
kite=kite2
kite=kite3
lamb=lamb1
mother=mother1
nana=nana1
nana=nana2
ostrich=ostrich1
pool=pool1
yak=yak1
yak=yak2
zebra=zebra1
-----------
keys and values:
apple apple1
cat cat1
dog dog1
elephant elephant1
frog frog4 frog3 frog2 frog1
girl girl1
house house1
infix infix1
jail jail1
kite kite3 kite2 kite1
lamb lamb1
mother mother1
nana nana2 nana1
ostrich ostrich1
pool pool1
yak yak2 yak1
zebra zebra1
----RESIZED HASH-------
keys and values:
apple apple1
cat cat1
dog dog1
elephant elephant1
frog frog4 frog3 frog2 frog1
girl girl1
house house1
infix infix1
jail jail1
kite kite3 kite2 kite1
lamb lamb1
mother mother1
nana nana2 nana1
ostrich ostrich1
pool pool1
yak yak2 yak1
zebra zebra1
